SpriterDotNet/SpriterParser.cs

```diff
@@ -21,8 +22,8 @@ static SpriterParser()
 
         public static Spriter Parse(string data)
         {
+            if (data != null) data = data.Trim();
```

Comment:
Wanted to, but IsNullOrWhiteSpace is a .Net 4.0 method and isn't available. You can set up your own extension method pretty easily, but might be overkill without using it anywhere else at the moment.